src/appMain/sdl_preloaded_pt.json

```diff
@@ -3334,13 +3334,17 @@
                     "type": "Float",
                     "mandatory": false,
                     "minvalue": -6,
-                    "maxvalue": 106
+                    "maxvalue": 106,
```

Comment:
@LitvinenkoIra @LuxoftAKutsan to check if we need ot here

Reply:
Need to create an issue to remove VD items from sdl_preloaded. 
Need to investigate if they a realy redundant  .

Reply:
Extract sdl_preloded_pt to separate commit with a comment that commit added for consistency,